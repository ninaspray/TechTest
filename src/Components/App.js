import React, { useState }from 'react';
import "../Styles/App.css";
import Search from './Search';
import SearchResults from './searchResults';


function App() {

const [searchResults, setSearchResults] = useState([]);

  return (
    <div className="App">
      <img
        className="logo"
        src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
        alt="NASA logo"
      />
      <Search setSearchResults={setSearchResults} />
      <SearchResults results={searchResults}/>
    </div>
  );
}

export default App;
