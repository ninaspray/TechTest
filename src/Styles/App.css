.App {
    display: flex;
    flex-direction: column;
  }
  
  .nasa-logo-link {
    margin: auto;
    width: 500px;
    height: 300px;
  }