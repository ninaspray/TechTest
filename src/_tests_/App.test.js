import React from 'react';
import { render, screen } from '@testing-library/react';
import App from '../Components/App';

describe ("App", () => {
    const {asFragment} = render (<App/>);

    it("renders App Correctly", () => {
        expect(asFragment()).toMatchSnapshot();
    });
});
