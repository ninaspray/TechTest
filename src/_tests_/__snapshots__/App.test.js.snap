// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders App Correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <img
      alt="NASA logo"
      class="logo"
      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
    />
    <div
      class="Search"
    >
      <form
        class="search-form"
      >
        <input
          class="Search-field"
          type="text"
        />
        <button
          class="search-comp"
          type="submit"
        >
          Search!
        </button>
      </form>
    </div>
    <p>
      Search Results
    </p>
    <img
      alt="space-pic"
      class="card-image"
      src="https://images.unsplash.com/photo-1522030299830-16b8d3d049fe?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80"
    />
  </div>
</DocumentFragment>
`;
